lib/light-service/context.rb

```diff
@@ -116,7 +116,7 @@ def stop_processing?
     end
 
     def define_accessor_methods_for_keys(keys)
-      return if keys.blank?
+      return if keys.empty?
```

Comment:
@adomokos since we are adding back `activesupport`, do we really need to make this change? Instead, we could simply include `active_support/core_ext/object/blank` and use the same method without introducing additional changes.

Reply:
The goal is to yank Active Support dependency, entirely, for pure Ruby projects. Not sure how easily feasible (or how much needed) that is, though.